.gitignore

```diff
@@ -23,3 +23,6 @@ vendor
 *.swp
 *.swo
 *~
+
+$ generated config files
```

Comment:
Seems like this should be a comment with a `#`

Reply:
Thanks for catching this typo!